java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java

```diff
@@ -0,0 +1,116 @@
+package com.plaid.quickstart.resources;
+
+import java.io.IOException;
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import com.plaid.client.PlaidClient;
+import com.plaid.client.request.paymentinitiation.PaymentCreateRequest;
+import com.plaid.client.request.paymentinitiation.PaymentGetRequest;
+import com.plaid.client.request.paymentinitiation.PaymentTokenCreateRequest;
+import com.plaid.client.request.paymentinitiation.RecipientCreateRequest;
+import com.plaid.client.response.ErrorResponse;
+import com.plaid.client.response.paymentinitiation.PaymentCreateResponse;
+import com.plaid.client.response.paymentinitiation.PaymentGetResponse;
+import com.plaid.client.response.paymentinitiation.PaymentTokenCreateResponse;
+import com.plaid.client.response.paymentinitiation.RecipientCreateResponse;
+import com.plaid.client.model.paymentinitiation.Address;
+import com.plaid.client.model.paymentinitiation.Amount;
+import com.plaid.quickstart.QuickstartApplication;
+
+import java.util.Arrays;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import retrofit2.Response;
+
+@Path("/payment_initiation")
+@Produces(MediaType.APPLICATION_JSON)
+public class PaymentInitiationResource {
+  private PlaidClient plaidClient;
+
+  public PaymentInitiationResource(PlaidClient plaidClient) {
+    this.plaidClient = plaidClient;
+  }
+
+  @GET
+  public PaymentGetResponse getPayment() throws IOException {
+    String paymentId = QuickstartApplication.paymentId;
+
+    Response<PaymentGetResponse> paymentGetResponse =
+      plaidClient.service().paymentGet(new PaymentGetRequest(paymentId)).execute();
+    if (!paymentGetResponse.isSuccessful()) {
+      try {
+        ErrorResponse errorResponse = plaidClient.parseError(paymentGetResponse);
+        Thread.sleep(3000);
```

Comment:
Why call `Thread.sleep`?

Reply:
We should not, it was copy-pasta https://github.com/plaid/quickstart/pull/90/commits/bedafdf1aed41d7b28f047ed8d004dc3ad1d3810